pkgs/by-name/ky/kyverno-chainsaw/package.nix

```diff
@@ -11,23 +11,23 @@
 
 buildGoModule rec {
   pname = "kyverno-chainsaw";
-  version = "0.2.8";
+  version = "0.2.11";
 
   src = fetchFromGitHub {
     owner = "kyverno";
     repo = "chainsaw";
     rev = "v${version}";
-    hash = "sha256-elszVinOGL4IbMNK3RTtzJPRRqs7qgqITQqj0g/wyk8=";
+    hash = "sha256-s66TG4r3WSmEfcJzibu8uKv1Pr0aKfTEFNfJzLHjbIA=";
   };
 
-  vendorHash = "sha256-HDmnI+WVv4cUia+IXrBdFBFnrxKMSFjKD6LUZ393sqs=";
+  vendorHash = "sha256-NQJbj1lFR5947Lrw5nSsdfYlkWjTWJ5ybMq7Bpd/s3I=";
+
+  subPackages = [ "." ];
 
   ldflags = [
     "-s"
     "-w"
     "-X github.com/kyverno/chainsaw/pkg/version.BuildVersion=v${version}"
-    "-X github.com/kyverno/chainsaw/pkg/version.BuildHash=${version}"
```

Comment:
It seems these extraneous values have been specified since [the beginning](https://github.com/NixOS/nixpkgs/pull/292144), although chainsaw has used [runtime/debug](https://pkg.go.dev/runtime/debug#BuildInfo) since at least v0.1.7 (see: https://github.com/kyverno/chainsaw/blob/v0.1.7/pkg/version/version.go)